test/e2e/framework/util.go

```diff
@@ -5163,3 +5191,17 @@ func WaitForPersistentVolumeClaimDeleted(c clientset.Interface, ns string, pvcNa
 	}
 	return fmt.Errorf("PersistentVolumeClaim %s is not removed from the system within %v", pvcName, timeout)
 }
+
+func GetClusterZones(c clientset.Interface) sets.String {
```

Comment:
We already have other e2e tests that get all the zones in the cluster.  Can you use the same api they're using instead?

Reply:
Talked to @msau42 offline. Most tests re-implement this functionality with the same way of getting zone info. I'm changing some of them to use this method instead.